### ISSUE TITLE
Warnings with ruby 2.7.0 and consul-templaterb 1.21.8

### ISSUE THREAD

**Issue by kshpytsya** [criteo/consul-templaterb#57]
```
/usr/local/bundle/gems/consul-templaterb-1.21.8/bin/consul-templaterb:291: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/local/bundle/gems/consul-templaterb-1.21.8/lib/consul/async/consul_endpoint.rb:11: warning: The called method `initialize' is defined here
/usr/local/bundle/gems/consul-templaterb-1.21.8/bin/consul-templaterb:292: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/local/bundle/gems/consul-templaterb-1.21.8/lib/consul/async/vault_endpoint.rb:15: warning: The called method `initialize' is defined here
```

**Comment by pierresouchay** [comment]
Fixed by https://github.com/criteo/consul-templaterb/commit/357b3cf748670950292820de17cc8179e586d30d

**Comment by pierresouchay** [comment]
fixed in version 1.22.0
